docs/migration_guide.md

```diff
@@ -8,6 +8,54 @@ Lisk Service 0.6.0 allows to export transactions in CSV format.
 
 To enable this feature make sure your PM2 config has the lisk-service-export microservice started. Docker script enables it automatically, however the gateway needs a param to enable it to the public.
 
-Consider adding the `http-exports` string to `ENABLE_HTTP_API` in order to make this feature usable by the clients.
+Consider adding the `http-exports` string to `ENABLE_HTTP_API` in order to make this feature available for users.
 
 `ENABLE_HTTP_API=http-status,http-version2,http-exports`
+
+### Blockchain index
+
+Starting from `0.6.0` you do not need to rebuild the index every release. This will save much time during migrations and allow minimal interruptions of the Lisk Service availabilty.
+
+To make a snapshot on the `0.5.0` instance the following steps are needed.
+
+```bash
+docker-compose stop core
+docker-compose exec -T mysql_core mysqldump --no-create-db lisk -u root -ppassword > mysql_core_index.sql
```

Comment:
```suggestion
make stop-core
make backup-db
```

Reply:
These commands don't exist in the 0.5.0 which we are migration from.